### PR TITLE
feat(savedaddresses): context menu - remove saved address

### LINKED ISSUE #13093: Context Menu - Remove saved address
Related figma:
https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=21991%3A256134&mode=dev

Implies:
- adding logic for the context menu "Remove saved address" action
- ephemeral notification for the action

### PR DESCRIPTION
**Pull request by saledjenic**
Closes #13093

### CODE REVIEW

File: ui/app/mainui/AppMain.qml [hunk 1]
```diff
@@ -1771,63 +1771,49 @@ Item {
...
             target: WalletStore.RootStore.walletSectionSavedAddressesInst
 
-            function onSavedAddressDeleted(address: string, ens: string, errorMsg: string) {
+            function onSavedAddressDeleted(name: string, address: string, ens: string, errorMsg: string) {
                 WalletStore.RootStore.deletingSavedAddress = false
                 WalletStore.RootStore.lastDeletedSavedAddress = { address: address, ens: ens, error: errorMsg }
+
+                if (!!errorMsg) {
+
+                    Global.displayToastMessage(qsTr("An error occurred while removing %1 address").arg(mode).arg(name),
```
Comment (caybro) [inline ui/app/mainui/AppMain.qml:1800]: You have 2 `args` here but only one placeholder (`%1`)

**Review by caybro — COMMENTED** [review]
Otherwise LGTM :)

**Review by alaibe — APPROVED** [review]

File: ui/app/mainui/AppMain.qml (hunk 1 above)
Comment (saledjenic) [reply ui/app/mainui/AppMain.qml]: Fixed, thanks :)

**Review by caybro — APPROVED** [review]
LGTM

**Pull request merged**